assignment-1/Failed__case.md

```diff
@@ -1,87 +1,86 @@
+
+The client were according to [4, page 6]: the Department of Health and Human Services (HHS), Centers for Medicare & Medicaid Services (CMS), 36 States, 300 private insurers, U.S. Chief Technology Office, GAO, Media, Citizens, Social Security Administration, the Internal Revenue Service, Veterans Administration, Office of Personnel Management, Peace Corps, etc.
+
+The internal architecture of healthcare.gov is very complex. US residents who want to apply for an insurance needed a lot of information from the different systems. Like their income and immigration status. The website needed to be connected with all the different systems from the federal government and the private insurers. Like the Internal Revenue Service, Social Security Administration, the Peace Corps, etcetera. The idea was the information of all the different federal systems should be retrieved real time. According to [4] the U.S. Chief Technology Officer Todd Park has said that the government expected that the government expected HealthCare.gov to draw 50,000 to 60,000 simultaneous users. But in the first week the site was overwhelmed by up to five times as many users. 
+
+#####Timeline of problems and solutions.
+
+######Before the launch
+There were a lot of warnings before the lauch of the website by different parties. But is seems nothing is done with them. Below are a couple of them noted
+
+On April 4, 2013 a 15-page document by McKinsey & Co. warns among others the CMS administrator that there was insufficient end-to-end testing and advises that a limited initial launch of the website would be ideal. It seems nothing is done with this report, because it surfaced only half November 2013. One and a half month after the lauch. Next to that there were only two weeks of end-to-end testing before the launch took place for all the users.
+
+The Government Accountability Office (GAO), an independent organisation of the government, reported in June 2013 that there could be trouble with the goal of launching the website on the first of October 2013. 
+
+On 10 September there was a panel of the House of Representatives who were hearing among others consulting firm Leavitt Partners and contractor CGI. CGI stated during this hearing that it was on schedule for launching the website. On the other hand Leavitt Partners stated that the launch will be “rocky” [4, page 7] because there are still ongoing technology challenges for the online health insurance exchange.
+
+According to the New York Times [4, page 7], documents released by House investigators reveal that the ‘testing bulletin’ showed that the website as of September 30 could handle only about 1100 users at a time, even though officials had said it should have been abble to accomodate perhaps as many as 60,000 users.
```

Comment:
September 30, 2013
According to the New York Times, documents released by House investigators reveal that the ‘testing bulletin’ suggested the website as of September 30 could handle only about 1100 users at a time, even though ofﬁcials had said it should have been able to accommodate perhaps as many as 60,000.

[Carla] yes, the officials really said 'perhaps'. About 'officials'...have/had nothing to do with plurals.have is for recent past, and had for past far away.